docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -1079,6 +1079,157 @@ These are not currently modeled.
+
+- The sample will be taken at set time intervals (monthly, annually) that the research team will communicate in terms of time steps 
+- The total duration of the survey will be specified 
+- Once there are stable addresses, the sample will be taken and kept the same at the address level. For now, when addresses are not stable, we can keep the same households instead. 
+
+
+**What to Sample** 
+
+.. list-table:: Simulant Attribute to Sample 
+  :widths: 20
+  :header-rows: 0
+
+  * - Unique simulant ID (for PRL tracking)
+  * - First name
+  * - Middle initial 
+  * - Last name
+  * - Age 
+  * - Home Address 
+
+**Who to Sample** 
+
+For surveys, there is a much more significant amount of non-response bias 
+compared to the annual census. Participation will be determined in a two 
+step process. 
+
+**Step 1:** Households will be randomly selected for participation at a rate 
+predetermined by the researcher. The selection should be stratified by state, 
+but no other variables. This will be a random sample. 
+
+**Step 2:** Households will be chosen to be non-responsonders and removed from 
+the sample. This step will vary significantly based on the mode of the survey. 
+There are three possible modes: mail/online (assumed to be the same for this 
+model), telephone, and personal visits. 
+
+The below table includes the percent of responses for each mode of survey by 
+race/ethnicity. These will be used to find the non-response based on the mode 
+of the survey. 
+
+
+.. list-table:: Simulant Response by Race/Ethnicity 
+  :widths: 20 10 10 10 
+  :header-rows: 1
+
+  * - Race/Ethnicity  
+    - Mail/Online Percent of Response 
+    - Telephone Percent of Response
+    - Personal Visit Percent of Response 
+  * - White 
+    - 62.5%
+    - 11.9%
+    - 25.6% 
+  * - Black 
+    - 29.7%
+    - 15.1%
+    - 55.2% 
+  * - Asian 
+    - 52.7%
+    - 9.7%
+    - 37.6%  
+  * - American Indian and Alaskan Native 
+    - 40.1% 
+    - 18.0% 
+    - 41.9% 
+  * - Native Hawaiian and Pacific Islander  
+    - 30.3%
+    - 14.2%
+    - 55.8% 
+  * - Other Races or Multiracial  
+    - 22.9% 
+    - 16.7% 
+    - 60.3% 
+  * - Hispanic/Latino 
+    - 25.9% 
+    - 15.1% 
+    - 60.3% 
+
+
+For surveys with mail, telephone, and personal visits, assume that 
+non-response bias matches the census, using the data in the section above. 
+
+.. list-table:: Calculating Simulant Non-Response 
+  :widths: 10 10 10 20
+  :header-rows: 1
+
+  * - Simulant 
+    - Race/Ethnicity 
+    - Survey Modes USed 
+    - Probability of Omission (%)
+  * - 1
+    - White 
+    - Mail/Online Only 
+    - 100% - 62.5% = 37.5% non-response 
+  * - 2
+    - Black  
+    - Mail/Online and Telephone  
+    - 100% - (29.7% + 15.1%) = 55.2% non-response 
+  * - 3
+    - Asian 
+    - Mail/Online, Telephone, and Personal Visits 
+    - Use census data above, dependent on age/sex
```

Comment:
I also think there is potential confusion here because you say "dependent on age/sex" but race/ethnicity is already specified to be Asian so I assume that effect is included as well. Even if you have to specify a particular age and sex, having a full example for this would be helpful.